Block/Adminhtml/Edit/Balance/DashboardLink.php

```diff
@@ -24,50 +22,104 @@ class DashboardLink extends Template
     protected $requestFactory;
 
     /**
+     * DashboardLink constructor.
      * @param Template\Context $context
      * @param SellerFactory $sellerModel
      * @param Config $balancepayConfig
      * @param RequestFactory $requestFactory
+     * @param SellerFactory $sellerFactory
      * @param array $data
      */
     public function __construct(
         Template\Context $context,
         SellerFactory $sellerModel,
         Config $balancepayConfig,
         RequestFactory $requestFactory,
+        SellerFactory $sellerFactory,
         array $data = []
     ) {
         $this->sellerModel = $sellerModel;
         $this->balancepayConfig = $balancepayConfig;
         $this->requestFactory = $requestFactory;
+        $this->sellerFactory = $sellerFactory;
         parent::__construct($context, $data);
     }
 
     /**
+     * Is Vendor Id Set
+     *
      * @return bool
      */
-    public function isVendorIdSet() {
-        return (bool) $this->getBalanceVendorId();
+    public function isVendorIdSet()
+    {
+        return (bool)$this->getBalanceVendorId();
     }
 
     /**
+     * Get Balance Pay Dashboard URL
+     *
      * @return string
      */
-    public function getBalancePayDashboardUrl() {
+    public function getBalancePayDashboardUrl()
+    {
         $vendorId = $this->getBalanceVendorId();
         $balancePayDashboardUrl = $this->balancepayConfig->getBalanceDashboardUrl();
-        return $balancePayDashboardUrl.'/vendors/'.$vendorId;
+        return $balancePayDashboardUrl . '/vendors/' . $vendorId;
     }
 
-    public function getBalanceVendor() {
+    /**
+     * Get balancepaystatus
+     *
+     * @return string
+     */
+    public function getBalancePayStatus()
+    {
+        $customerId = $this->getRequest()->getParam('id');
+        $collection = $this->sellerModel->create()
+            ->getCollection()
+            ->addFieldToFilter('seller_id', $customerId)
+            ->addFieldToFilter('is_seller', 1);
+        if (isset($collection) && count($collection) > 0) {
+            foreach ($collection as $col) {
+                if ($col->getPayouts()) {
+                    return 'Enabled';
```

Comment:
I think you could use enum to represent the status, to avoid string comparison issues